packages/core/src/indexing/postgres/postgres-index-api.ts

```diff
@@ -87,7 +87,7 @@ export class PostgresIndexApi implements DatabaseIndexApi {
       )
       .onConflict('model')
       .merge({
-        updated_at: this.dbConnection.fn.now(),
+        updated_at: new Date(), // we don't use this.dbConnection.fn.now(), because postgres datetime may have higher precision than js date; TODO: CDB-XXX: set postgres created_at and updated_at precision to 3
```

Comment:
```suggestion
        updated_at: new Date(), // we don't use this.dbConnection.fn.now(), because postgres datetime may have higher precision than js date; TODO: CDB-2006: set postgres created_at and updated_at precision to 3
```